metrics/web/test/features/common/presentation/scaffold/widget/metrics_scaffold_test.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/features/common/presentation/app_bar/widget/metrics_app_bar.dart';
+import 'package:metrics/features/common/presentation/scaffold/widget/metrics_scaffold.dart';
+
+void main() {
+  group("MetricsScaffold", () {
+    const drawer = Drawer();
+
+    testWidgets(
+      "thows an AssertionError if trying to create without body",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(const _MetricsScaffoldTestbed(body: null));
+
+        expect(tester.takeException(), isAssertionError);
```

Comment:
Maybe use this `isAssertionError` in other tests where possible (for example, in tests for the `MetricsTheme`)?